frontend/app/views/info/supporterAustralia.scala.html

```diff
@@ -7,13 +7,20 @@
 @import com.gu.memsub.subsv2.Catalog._
 @import com.gu.memsub.subsv2.MonthYearPlans
 @import views.support.MembershipCompat._
+@import com.gu.salesforce.Tier.Supporter
+@import tracking.AppnexusPixel
 
 @import com.gu.memsub.subsv2.CatalogPlan
 @(heroImage: model.OrientatedImages, supporterPlans: MonthYearPlans[CatalogPlan.Supporter],
   pageInfo: PageInfo,
   pageImages: Seq[com.gu.memsub.images.ResponsiveImageGroup])(implicit token: play.filters.csrf.CSRF.Token, countryGroup: CountryGroup)
 
 @main("Supporters", pageInfo=pageInfo, countryGroup=Some(countryGroup)) {
+
+    <!-- Segment Pixel - Jan17-Membership Landing Page - DO NOT MODIFY -->
+        <img src="https://secure.adnxs.com/px?add=@{AppnexusPixel.landingPageids(Supporter())}&t=2" width="1" height="1" />
```

Comment:
if this url is the same everywhere except for the Tier, I would try to extract its generation to a single place, like a fragment or something.
But if there's no time for that now I'm happy for it to be shipped like this and we can leave it for after the holidays